src/main/kotlin/com/terraformation/backend/email/EmailService.kt

```diff
@@ -138,9 +136,23 @@ class EmailService(
    * @param [model] Model object containing values that can be referenced by the template.
    */
   fun sendSupportNotification(model: EmailTemplateModel) {
-    config.support.email?.let { supportEmail ->
-      Locale.ENGLISH.use { send(model, listOf(supportEmail)) }
-    }
+    config.support.email?.let { supportEmail -> sendLocaleEmails(listOf(supportEmail), model) }
+  }
+
+  /**
+   * Sends an email notification to a specific list of emails, using the specified Locale.
+   *
+   * @param [recipients] Email addresses to send the message to.
+   * @param [model] Model object containing values that can be referenced by the template.
+   * @param [initialLocale] Optional Locale object; uses ENGLISH if unspecified.
+   */
+  fun sendLocaleEmails(
+      recipients: List<String>,
+      model: EmailTemplateModel,
```

Comment:
Maybe flip the order of `recipients` and `model` so it matches the order of `send()`'s arguments.